util/coverage/coverage.js

```diff
@@ -369,7 +369,7 @@ document.addEventListener('DOMContentLoaded', () => {
   // Allow for `coverage.html?lang=de` style constructions.
   const params = new URLSearchParams(window.location.search);
   const lang = params.get('lang') ? params.get('lang') : 'en';
-  document.body.classList.add(`lang-${lang}`);
+  document.documentElement.lang = lang === 'cn' ? 'zh-CN' : lang;
```

Comment:
What about a shared function that this and `UserConfig` can call to get the right mapping from cactbot lang to locale, possibly in `resources/languages.ts`?

I think you missed a few, as well:
```
$ git grep lang- -- *.css
resources/defaults.css:.lang-ja .text {
resources/defaults.css:.lang-cn .text {
ui/raidboss/raidboss.css:#timeline-container.lang-fr {
ui/raidboss/raidboss.css:#timeline-container.lang-de {
ui/raidboss/raidboss.css:#timeline-container.lang-ko {
ui/raidboss/raidboss.css:.lang-ko .timer-bar {
```

Reply:
Oh, sure!
I'll take a look later.

Reply:
Also thinking about this,l this is a breaking change if users are depending on `lang-cn` existing as a style.  I think we could either keep the old CSS as well or alternatively I could write a release note for this.

Reply:
True.
Maybe I can say something like
```css
:lang(zh-CN) .text,
/* old one */
.lang-cn .text {
  /* blahblah */
}
```